docs/zotero.html

```diff
@@ -261,8 +265,8 @@ <h2>Zotero Library Shout-Outs</h2>
             });
             
             // Source Verification Quiz Logic
-            document.querySelectorAll('#quiz-source .quiz-option').forEach(btn = > {
-                btn.addEventListener('click', () = > {
+            document.querySelectorAll('.quiz-option').forEach(btn => {
```

Comment:
@dal5842 Well, I did change this but it might not be what you want. I don't know if querySelectorAll will do parent child selection like this and I wondered if just setting it to the class quiz-option on its own would work. It didn't...something else is wrong here.